src/frontend/src/lib/utils/certified-store.utils.ts

```diff
@@ -0,0 +1,5 @@
+import type { CertifiedData } from '$lib/types/store';
+import type { Option } from '$lib/types/utils';
+
+export const mapCertifiedData = <T>(certifiedData: Option<CertifiedData<T>>): Option<T> =>
```

Comment:
It's a trivial but important function. Can you add a small test given it's a utils now?

Reply:
right! forgot to commit it, sorry